.github/workflows/03-post-release.yaml

```diff
@@ -15,3 +15,21 @@ jobs:
         with:
           hash-type: sha1
           file-name: kubescape-release-digests
+      - name: Invoke workflow to update packaging
```

Comment:
@HollowMan6 We should invoke this only if the repository owner is kubescape.

Reply:
Done! @dwertent